.gitignore

```diff
@@ -34,4 +34,5 @@ _site/
 test/benchmark/e2e/http/results/
 tmp/
 test/coverage.tmp
-
+protocol/kafka_sarama/v2/vendor
+protocol/kafka_segmentio/v2/vendor
```

Comment:
why?

Reply:
perhaps the author ran `go mod vendor`? I agree, we don't need to commit the vendor dir nor ignore it.